app/scripts/services/validateRegistrant.js

```diff
@@ -30,20 +41,24 @@ angular.module('confRegistrationWebApp')
         }
       }
 
-      return block.rules.length === 0 || // If no rules are set
+      return blockTypeSpecificRules.length === 0 || // If no rules are set
```

Comment:
FYI you'll probably have a merge conflict with master. I added `!block.rules ||` to master because we ran into an issue where blocks weren't showing up. It was hard to reproduce but I thought block.rules might not be set in some cases. Here I think it'd be good if you add `!blockTypeSpecificRules ||` to the beginning in case it isn't defined or isn't an array or something. Your filters above might automatically create an array. Idk.